job-service-db/src/main/resources/v2.3.0/changelog.xml

```diff
@@ -21,6 +21,21 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
 
+	<changeSet id="update_job_table_add_column_job_dependency_change" author="Connor Mulholland">
+		<addColumn tableName="job">
+			<column name="delay" type="INT" defaultValueNumeric="0">
+				<constraints nullable="true"/>
```

Comment:
It's probably just something I don't know about Liqibase but it seems strange to say nullable=true and then add a not-null constraint.

Reply:
Not its a good question. I initially done it this way to accommodate database upgrade of existing rows. I will try something else here though.